featurewiz/my_encoders.py

```diff
@@ -56,7 +56,6 @@ def __init__(self):
     def fit(self,testx, y=None):
         ### Do not change this since Rare class combiner requires this test ##
         if isinstance(testx, tuple):
-            y = testx[1]

```

Comment:
No need in fit .

Reply:
You have made a mistake in below code:
```
test_result = MLB.fit_transform(train[everycol])
```
Instead it should be:
```
test_result = MLB.transform(test[everycol])
```
Do you see the difference?

Reply:
Right , my mistake , now fixed